docs/modules/ROOT/pages/getting-started/roles_persmissions.adoc

```diff
@@ -1,20 +1,20 @@
 = Permissions and roles
 
-As a workspace owner, you can https://placeholder.com[invite] one or more collaborators to your workspace by adding their user ID, such as an external SSO ID. After https://placeholder.com[adding a user to a workspace], you assign them a specific role that determines the actions they can take in a workspace.
+As a workspace owner, you can invite collaborators to your workspace by adding their user ID, such as an external SSO ID. After adding a user to a workspace, you assign them a specific role that determines the actions they can take in that workspace.
```

Comment:
It's not like you can really "invite" anyone. You are adding a user. The user you are adding can't opt-out. So not a real invitation.

Also, we only support adding usernames. User ID can mean different things.